CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Changes to Calva.
 
 ## [Unreleased]
 
+- [LSP support for multi-project and multi-workspace](https://github.com/BetterThanTomorrow/calva/pull/2020)
+- Fix: [Clojure-lsp caching in multi-project workspace](https://github.com/BetterThanTomorrow/calva/issues/934)
+- Fix: [With multi-root workspaces the clojure-lsp project root is always the first folder](https://github.com/BetterThanTomorrow/calva/issues/1706)
```

Comment:
Great that you've found these! Maybe @bpringe and I can help dig around more and see if we are catching more birds with this stone?

Reply:
Anything you want me to test in particular?

Reply:
Oh, I meant dig in the issue pile. Maybe there is more that this PR is closing.

Reply:
Looked around some but did not see others.